CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+
+* **Overall**
+
+* **Metrics**
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 43.3.1 to 45.28.0
+  * [UPGRADE] - Prometheus has been upgraded from version 2.40.7 to 2.44.0
+  * [UPGRADE] - Prometheus Operator has been upgraded from version 0.62.0 to 0.65.1
+  * [UPGRADE] - Grafana has been upgraded from version 9.3.1 to 9.5.2
+  * [UPGRADE] - Kube State Metrics has been upgraded from version 2.7.0 to 2.8.2
+  * [UPGRADE] - Pushgateway has been upgraded from version 1.4.3 to 1.5.1
```

Comment:
According to the image inventory and CHANGELOG, we moved to Pushgateway version 1.5.1 in our 1.2.11 release.

Reply:
Good catch.  I copied some lines and forgot to update it.  This has been correctly updated to the Pushgateway Helm chart with the appropriate version number (if we feel like this is something that we need to note).